Sources/AVPlayerLayer+Android.swift

```diff
@@ -31,15 +32,13 @@ public class AVPlayerLayer: JNIObject {
     public var frame: CGRect {
         get { return .zero } // FIXME: This would require returning a JavaObject with the various params
         set {
-            Task { @MainActor in
-                let scaledFrame = (newValue * UIScreen.main.scale)
-                try! call(methodName: "setFrame", arguments: [
-                    JavaInt(scaledFrame.origin.x.rounded()),
-                    JavaInt(scaledFrame.origin.y.rounded()),
-                    JavaInt(scaledFrame.size.width.rounded()),
-                    JavaInt(scaledFrame.size.height.rounded())
-                ])
-            }
+            let scaledFrame = (newValue * UIScreen.main.scale)
+            try! call(methodName: "setFrame", arguments: [
+                JavaInt(scaledFrame.origin.x.rounded()),
+                JavaInt(scaledFrame.origin.y.rounded()),
+                JavaInt(scaledFrame.size.width.rounded()),
+                JavaInt(scaledFrame.size.height.rounded())
+            ])
```

Comment:
this change is unrelated to the crash, but I thought it's easier to flag the AVPlayerLayer with @mainactor and get rid of the task